django/utils/http.py

```diff
@@ -163,7 +163,9 @@ def parse_http_date(date):
     try:
         year = int(m.group('year'))
         if year < 100:
-            if year < 70:
+            current_year = datetime.date.today().year - 2000  # 2018 -> 18
```

Comment:
I haven't read the RFC but I assume we want to be using UTC's today? Right now this using the server's timezone today.

Reply:
@tameeshB any thoughts about the above comment?